test-junit5/src/main/java/io/micronaut/test/extensions/junit5/annotation/MicronautTest.java

```diff
@@ -42,6 +43,7 @@
 @Factory
 @Inherited
 @Requires(condition = TestActiveCondition.class)
+@Executable
```

Comment:
It would be more efficient if we made `@Test` == `@Executable` but since this is test code I think it is probably ok